spring-cloud-starter-stream-sink-pgcopy/src/main/java/org/springframework/cloud/stream/app/pgcopy/sink/PgcopySinkConfiguration.java

```diff
@@ -0,0 +1,251 @@
+
+import org.postgresql.copy.CopyIn;
+import org.postgresql.copy.CopyManager;
+import org.postgresql.core.BaseConnection;
+
+import org.springframework.beans.factory.FactoryBean;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.cloud.stream.annotation.EnableBinding;
+import org.springframework.cloud.stream.app.jdbc.DefaultInitializationScriptResource;
+import org.springframework.cloud.stream.binding.InputBindingLifecycle;
+import org.springframework.cloud.stream.messaging.Sink;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Primary;
+import org.springframework.core.io.ResourceLoader;
+import org.springframework.dao.DataAccessException;
+import org.springframework.integration.aggregator.DefaultAggregatingMessageGroupProcessor;
+import org.springframework.integration.aggregator.ExpressionEvaluatingCorrelationStrategy;
+import org.springframework.integration.aggregator.MessageCountReleaseStrategy;
+import org.springframework.integration.annotation.ServiceActivator;
+import org.springframework.integration.channel.DirectChannel;
+import org.springframework.integration.config.AggregatorFactoryBean;
+import org.springframework.integration.store.MessageGroupStore;
+import org.springframework.integration.store.MessageGroupStoreReaper;
+import org.springframework.integration.store.SimpleMessageStore;
+import org.springframework.jdbc.core.ConnectionCallback;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.datasource.init.DataSourceInitializer;
+import org.springframework.jdbc.datasource.init.ResourceDatabasePopulator;
+import org.springframework.jdbc.support.nativejdbc.Jdbc4NativeJdbcExtractor;
+import org.springframework.messaging.Message;
+import org.springframework.messaging.MessageChannel;
+import org.springframework.messaging.MessageHandler;
+import org.springframework.messaging.MessagingException;
+import org.springframework.scheduling.annotation.EnableScheduling;
+import org.springframework.scheduling.annotation.Scheduled;
+
+/**
+ * Configuration class for the PostgreSQL CopyManager.
+ *
+ * @author Thomas Risberg
+ */
+@Configuration
+@EnableScheduling
+@EnableBinding(Sink.class)
+@EnableConfigurationProperties(PgcopySinkProperties.class)
+public class PgcopySinkConfiguration {
+
+	private static final Log logger = LogFactory.getLog(PgcopySinkConfiguration.class);
+
+	@Autowired
+	private PgcopySinkProperties properties;
+
+	@Bean
+	public MessageChannel toSink() {
+		return new DirectChannel();
+	}
+
+	@Bean
+	@Primary
+	@ServiceActivator(inputChannel= Sink.INPUT)
+	FactoryBean<MessageHandler> aggregatorFactoryBean(MessageChannel toSink, MessageGroupStore messageGroupStore) {
```

Comment:
Merged? 
Why is this left even without comment?

What the policy do you apply to PRs if that another my about docs are ignored?

I wouldn't spend time on the task knowing that it will be ignored and now should be just closed as superseded...

Let me know what to read to understand how properly to proceed with task selections do not waste time in the future!

Thanks